tokio/src/io/driver/mod.rs

```diff
@@ -72,6 +72,7 @@ pub(super) struct Inner {
     io_dispatch: RwLock<IoDispatcher>,
 
     /// Used to wake up the reactor from a call to `turn`.
+    #[cfg(not(target_os = "wasi"))]
     waker: mio::Waker,
```

Comment:
This needs a comment explaining why it doesn't exist on WASM.

Reply:
Comment added